modules/storing-data-with-data-science-pipelines.adoc

```diff
@@ -0,0 +1,35 @@
+:_module-type: REFERENCE
+
+[id="storing-data-with-data-science-pipelines_{context}"]
+= Storing data with data science pipelines
+
+[role="_abstract"]
+When you run a data science pipeline, {productname-short} stores the pipeline YAML configuration file and resulting pipeline run artifacts in your storage bucket `root` directory. The directories that contain pipeline run artifacts can differ depending on where you executed the pipeline run from. See the following table for further information:
+
+.Pipeline configuration file and artifacts storage locations
+[cols="3]
+|===
+| Pipeline run source | Pipeline storage directory | Run artifacts storage directory
+
+| Ran from the {productname-short} dashboard
```

Comment:
Could probably remove 'Ran from the' from this line and line 22 without sacrificing any clarity.

Reply:
agreed